tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -5747,6 +5753,132 @@ SpirvEmitter::processTextureSampleCmp(const CXXMemberCallExpr *expr) {
       expr->getCallee()->getLocStart(), expr->getSourceRange());
 }
 
+SpirvInstruction *
+SpirvEmitter::processTextureSampleCmpBias(const CXXMemberCallExpr *expr) {
+  // .SampleCmpBias() Signature:
+  //
+  // For Texture1D, Texture1DArray, Texture2D, Texture2DArray:
+  // float Object.SampleCmpBias(
+  //   SamplerComparisonState S,
+  //   float Location,
+  //   float Bias,
+  //   float CompareValue
+  //   [, int Offset]
+  //   [, float Clamp]
+  //   [, out uint Status]
+  // );
+  //
+  // For TextureCube and TextureCubeArray:
+  // float Object.SampleCmpBias(
+  //   SamplerComparisonState S,
+  //   float Location,
+  //   float Bias,
+  //   float CompareValue
+  //   [, float Clamp]
+  //   [, out uint Status]
+  // );
+
+  const auto numArgs = expr->getNumArgs();
+  const bool hasStatusArg =
+      expr->getArg(numArgs - 1)->getType()->isUnsignedIntegerType();
+  auto *status = hasStatusArg ? doExpr(expr->getArg(numArgs - 1)) : nullptr;
```

Comment:
Could you process the arguments in order? It would be easier to follow.

Reply:
Done.